coderd/templates.go

```diff
@@ -84,10 +85,11 @@ func (api *API) deleteTemplate(rw http.ResponseWriter, r *http.Request) {
 		template          = httpmw.TemplateParam(r)
 		auditor           = *api.Auditor.Load()
 		aReq, commitAudit = audit.InitRequest[database.Template](rw, &audit.RequestParams{
-			Audit:   auditor,
-			Log:     api.Logger,
-			Request: r,
-			Action:  database.AuditActionDelete,
+			Audit:            auditor,
+			Log:              api.Logger,
+			Request:          r,
+			Action:           database.AuditActionDelete,
+			AdditionalFields: json.RawMessage("{}"),
```

Comment:
I think I would prefer if we checked if this was nil in the `audit.InitRequest` so we didn't have to provide this default in each handler!